docs-next/src/content/docs/running/editor-plugins.mdx

```diff
@@ -0,0 +1,44 @@
+---
+description: Configuring how Mocha finds test files.
+title: Editor Plugins
+---
+
+The following editor-related packages are available:
+
+### TextMate
+
+The [Mocha TextMate bundle](https://github.com/mochajs/mocha.tmbundle) includes snippets to make writing tests quicker and more enjoyable.
+
+### JetBrains
+
+[JetBrains](https://www.jetbrains.com) provides a [NodeJS plugin](https://plugins.jetbrains.com/plugin/6098-node-js) for its suite of IDEs (IntelliJ IDEA, WebStorm, etc.), which contains a Mocha test runner, among other things.
+
+![JetBrains Mocha Runner Plugin in Action](./jetbrains-plugin.png)
+
+The plugin is titled **NodeJS**, and can be installed via **Preferences** > **Plugins**, assuming your license allows it.
+
+### Wallaby.js
+
+[Wallaby.js](https://wallabyjs.com) is a continuous testing tool that enables real-time code coverage for Mocha with any assertion library in VS Code, Atom, JetBrains IDEs (IntelliJ IDEA, WebStorm, etc.), Sublime Text and Visual Studio for both browser and node.js projects.
+
+![Wallaby.js in Action](./wallaby.png)
+
+### Emacs
+
+[Emacs](https://www.gnu.org/software/emacs) support for running Mocha tests is available via a 3rd party package [mocha.el](https://github.com/scottaj/mocha.el).
+The package is available on MELPA, and can be installed via `M-x package-install mocha`.
+
+![Emacs Mocha Runner in Action](./emacs.png)
+
+### Mocha Sidebar (VS Code)
+
+[Mocha sidebar](https://marketplace.visualstudio.com/items?itemName=maty.vscode-mocha-sidebar) is the most complete mocha extension for vs code.
+
```

Comment:
This guy was last updated in 2019, for now I think no recommendation is better than a bad recommendation. Personally I use VS Code without any Mocha-specific extensions so I don't have a replacement offhand

```suggestion
```

Reply:
I think this would be a good thing to do in a separate change. As with the other content changes:

* Discussion: I know there are other extensions out there, and we should probably discuss more deeply which -if any- should get the "official" mochajs.org blessing
* Trackability: so it's easier to see where things got CRUD-ed

Reply:
Ref #5311